genologics/lims.py

```diff
@@ -562,8 +564,19 @@ def get_batch(self, instances, force=False):
             data = self.tostring(ElementTree.ElementTree(root))
             root = self.post(uri, data)
             for node in root.getchildren():
-                instance = instance_map[node.attrib['limsid']]
+                uri = node.attrib['uri']
```

Comment:
Exactly. And I wanted to do that because originally I wanted to fetch both versions of each artifact as you did, so we could compare them. (I later decided not to do the comparison, which we're discussing in another place.)